src/demo/demoUtils.js

```diff
@@ -30,6 +30,7 @@ export const getInitSdkOptions = () => {
       type:'document',
       options: {
         useLiveDocumentCapture: queryParamToValueString.useLiveDocumentCapture === 'true',
+        uploadFallback: queryParamToValueString.uploadFallback !== 'false',
```

Comment:
So uploadFallback is also available documents now? Should we add it to the documentation?

Reply:
Yes, it is. & good point. Thought I'd added it already, but will make sure it's documented in README.